docs/io/visualization/using_widgets.rst

```diff
@@ -178,4 +178,40 @@ There are also several other options in the modebar which we have not explained
 you remember to click back on the **Box Select** option for making selections on
 spectrum.
 
+Energy Level Diagram
+################
+
+This widget lets you visualize the last line interactions
+
+.. image:: ../images/energy_level_widget_demo.gif
+    :alt: Demo of Energy Level Diagram
+
+By selecting an ion on the widget, you can see its energy level diagram, which
+also shows information about the last line interactions experienced by packets
+in the simulation.
+
+The y-axis of the plot represents energy while the horizontal lines show
+discrete energy levels. The thickness of each line represents the level
+population, with thicker lines representing a greater population than the thin lines.
+
+Arrows represent the line interactions experienced by packets. Upwards arrows
+show excitation from lower energy levels to higher levels and downward arrows
+show de-excitation from higher energy levels to lower levels. The thickness of
+each arrow represents the number of packets that underwent that interaction,
+with thicker lines representing more number of packets than the thin lines.
```

Comment:
"more number of packets" -> "more packets"

Reply:
done